src/lib/cat_stories/index.svelte

```diff
@@ -328,18 +437,28 @@
       font-size: 18px;
     }
 
+    p {
+      text-align: center;
+    }
+
     .root {
       padding: 16px;
```

Comment:
Could you remove this padding?  The Figma has the component filling the entire screen from edge-to-edge.